pages/dao/[symbol]/params/GovernanceConfigModal.tsx

```diff
@@ -18,93 +18,173 @@ import useQueryContext from '@hooks/useQueryContext'
 import { useRouter } from 'next/router'
 import { notify } from '@utils/notifications'
 import useRealm from '@hooks/useRealm'
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
 import Button from '@components/Button'
 import BaseGovernanceForm, {
-  BaseGovernanceFormFields,
+  BaseGovernanceFormFieldsV2,
 } from '@components/AssetsList/BaseGovernanceForm'
-import { getGovernanceConfig } from '@utils/GovernanceTools'
+import { getGovernanceConfigFromV2Form } from '@utils/GovernanceTools'
 import {
   getDaysFromTimestamp,
   getMintDecimalAmountFromNatural,
 } from '@tools/sdk/units'
 import { abbreviateAddress } from '@utils/formatting'
 import * as yup from 'yup'
 import { DISABLED_VOTER_WEIGHT } from '@tools/constants'
+import useProgramVersion from '@hooks/useProgramVersion'
+import {
+  BaseGovernanceFormFieldsV3,
+  transformerGovernanceConfig_2_BaseGovernanceFormFieldsV3,
+  transform,
+  transformerBaseGovernanceFormFieldsV3_2_GovernanceConfig,
+} from '@components/AssetsList/BaseGovernanceForm-data'
 
-interface GovernanceConfigForm extends BaseGovernanceFormFields {
+type ProposalInfo = {
   title: string
   description: string
 }
 
+type GovernanceConfigFormV2 = BaseGovernanceFormFieldsV2 & ProposalInfo
```

Comment:
brief aside: the only difference (for the compiler) between types and interfaces is that interfaces are mutable and types aren't. I always just use types.